beacon-chain/execution/testing/BUILD.bazel

```diff
@@ -4,7 +4,6 @@ go_library(
     name = "go_default_library",
     testonly = True,
     srcs = [
-        "genesis.go",
```

Comment:
I had to move this out of the testing package, because there is a transitive `testonly` tag that prevents anything outside of tests from using code in this package.